samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/ble_mesh.c

```diff
@@ -33,7 +33,7 @@ static void prov_reset(void)
 	bt_mesh_prov_enable(BT_MESH_PROV_ADV | BT_MESH_PROV_GATT);
 }
 
-static u8_t dev_uuid[16];
+static u8_t dev_uuid[16] = { 0xdd, 0xdd };
```

Comment:
Is this in case bt_le_oob_get_local() fails, so you don't end-up with an all-zeroes UUID in that case? If so, please state it in the commit message. (I wouldn't call it "random" btw. Arbitrary, perhaps).

Reply:
bt_le_oob_get_local() is working properly after yesterday's patch from your side.
I just set dev_uuid[16] to { 0xdd, 0xdd } to mimic onoff-app.